changelogs/fragments/184-scrub-none-params.yaml

```
@@ -0,0 +1,4 @@
+---
+minor_changes:
+  - module_utils/core - Add helper function scrub_none_parameters to remove params set to None. Enabler
+    to fix issue (https://github.com/ansible-collections/community.aws/issues/251)
```

```suggestion
  - module_utils/core - add helper function ``scrub_none_parameters`` to remove params set to ``None`` (https://github.com/ansible-collections/community.aws/issues/251).
```
I removed the `Enabler to fix issue` part because IMO it is hard to understand, and not really needed because the link indicates this is related to the issue.